### DISCUSSION TITLE
day 17

### DISCUSSION THREAD

**Discussion by Junnyjun** [I-JUNNYLAND-I/algorithm#99] [Q&A] [answered]
[MANG JOO]
#97 #95 

Q[2]

제가 생각하는건 첫번째 문자열을 잘라서 가지고 있고
자른 문자가 다른 모든 문자열에 있으면 리스트에 담고 다른 모든 문자열에 해당 문자를 삭제해야된다?
라고 생각했는데 접근 방법을 바꿔야 할까요?

**Comment by Junnyjun** [comment, marked as answer]
중복 문자를 찾은 후 중복을 검사하면 됩니다.

A B C D E  -> A,1 B,1 C,1 D,1 E,1

B C D E F  -> B,2 C,2 D,2 E,2 ~F,1~

C D E F G  -> C,3 D,3 E,3 ~F,2~ ~G,1~

C,D,E 만 나오게됩니다.

만약 중복이 된다면 Map[ index ] > leng.size 일테니
(int) Map [ index ] / 2  로 나열해주시면 됩니다

**Reply by Mang-Joo** [reply]
답변 감사합니다 해당 로직으로 더 생각해서 풀어보겠습니다!
